learnr_bank/learnr_set3.Rmd

```diff
@@ -0,0 +1,338 @@
+---
+title: "Question Template Set 3"
+author: "Stephanie J. Spielman"
+date: "6/24/2020"
+output: learnr::tutorial
+runtime: shiny_prerendered
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(learnr)
+library(tibble)
+library(dplyr)
+```
+
+<!-- ALL MULTPLE CHOICE!!!!!! -->
+
+### Set 1: Understanding types of data I
+
+<!-- The first FIVE questions all have the same a/b/c/d choices. below each question i write which is correct -->
+a) Numeric discrete 
+b) Numeric continuous
+c) Categorical nominal
+d) Categorical ordinal
+d) Not enough information is provided
+ <!-- just one overall question -->
+### Determine what type of variable/data each of the following is. 
+In addition, with your group, come up with THREE different potential values for the variable. Use the example to help guide you:
+
+
+Example: **Square footage of a building** is *numeric continuous*. Example values are 1000, 5000, 300.5 - aka, random numbers which could be a building's square footage. I would not include a value -100, for example.
+
+``` {r quiz1, echo = FALSE}
+
+quiz(caption = "Multiple Choice Set 1",
```

Comment:
Hi Dr. Spielman, I was wondering if you could look over this line in particular? Apparently I have a bug, but I have been looking at it for a while and can't seem to find it.